build/vnu-jar.js

```diff
@@ -32,7 +32,8 @@ execFile('java', ['-version'], (error, stdout, stderr) => {
     // and offer more robust alternatives, but also need to show a less-than-ideal example
     'An “aria-disabled” attribute whose value is “true” should not be specified on an “a” element that has an “href” attribute.',
     // Boosted mod: `role="img"` is needed for `<img "src=.svg" alt="">`
-    'The “img” role is unnecessary for element “img”.'
+    'The “img” role is unnecessary for element “img”.',
```

Comment:
I do not think that this file should be included in the commit.

Reply:
This one is due to an issue that we had, having too many headers inside the documentation pages without being able to order those titles. Indeed, we display a GH (which have `<h1>`titles) inside a `<h3>` architecture and so, we're having trouble with a11y. As GH are not supposed to be in the middle of the page, we thought that it was the best solution so far.

Reply:
Could you just add a `// Boosted mod:` in order to explain why it is needed because we will forget the exact reason in few months.